main.js

```diff
@@ -0,0 +1,12 @@
+function daysInYear(year) {
+    try {
+        if (Number.isInteger(year) === false) {
+            throw  'exception';
+        }
+        return year % 400 === 0 || (year % 100 !== 0 && year % 4 === 0) ? 366 : 365;
+    } catch (e) {
+        console.log(e);
```

Comment:
Based on requirements: `it should throw an exception`. This line does not throw the excfeption.